src/main/java/com/google/devtools/build/lib/authandtls/AuthAndTLSOptions.java

```diff
@@ -180,7 +180,7 @@ public class AuthAndTLSOptions extends OptionsBase {
       documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
       effectTags = {OptionEffectTag.UNKNOWN},
       help =
-          "The duration for which credentials supplied by a credential helper are cached.\n\n"
+          "The default duration for which credentials supplied by a credential helper are cached if the helper does not provide when the credentials expire.\n\n"
               + "Invoking with a different value will adjust the lifetime of preexisting entries;"
               + " pass zero to clear the cache. A clean command always clears the cache, regardless"
```

Comment:
We also need to remove the sentence beginning with "Invoking with a different value ...", right? The `clean` command is now the only reliable way to clear the cache (which is probably fine).

Reply:
Yes. Dropped that sentence